# Announces a task to the robots.

uint32 request_id
uint32[] start_ids
geometry_msgs/Point[] start_points
uint32[] end_ids
geometry_msgs/Point[] end_points
